sdcm/stress_thread.py

```diff
@@ -119,10 +119,10 @@ def create_stress_cmd(self, node, loader_idx, keyspace_idx):
                 ' -transport "truststore=/etc/scylla/ssl_conf/client/cacerts.jks truststore-password=cassandra"'
 
         if self.node_list and '-node' not in stress_cmd:
-            first_node = [n for n in self.node_list if n.dc_idx == loader_idx %
-                          3]  # make sure each loader is targeting on datacenter/region
-            first_node = first_node[0] if first_node else self.node_list[0]
-            stress_cmd += " -node {}".format(first_node.cql_ip_address)
+            nodes = [n for n in self.node_list if n.dc_idx == loader_idx %
+                     3]  # make sure each loader is targeting on datacenter/region
+            nodes = nodes or self.node_list
+            stress_cmd += ' -node {}'.format(','.join([node.ip_address for node in nodes]))
```

Comment:
this change we should do for all of the stress threads, i.e. we should always select all known nodes